### PR TITLE
change  the module installed from git repos in requirements.txt

### PR DESCRIPTION
**Pull request by vinay-ebi**

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -1,4 +1,4 @@
-git+git://github.com/Ensembl/ensembl-prodinf-core.git#egg=ensembl-prodinf-core
+ensembl-prodinf-core@git+http://github.com/Ensembl/ensembl-prodinf-core.git#egg=ensembl-prodinf-core
```
Comment (marcoooo) [inline requirements.in:1]: HTTPS!

File: requirements.txt [hunk 2]
```diff
@@ -72,7 +70,7 @@ elasticsearch==6.8.2
     # via -r requirements.in
 email-validator==1.1.2
     # via -r requirements.in
-git+git://github.com/Ensembl/ensembl-prodinf-core.git#egg=ensembl-prodinf-core
+git+http://github.com/Ensembl/ensembl-prodinf-core.git#egg=ensembl-prodinf-core
```
Comment (marcoooo) [inline requirements.txt:73]: Module name + https

**Review by marcoooo — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (vinay-ebi) [reply requirements.txt:73]: i am using `pip-compile requirements.in -o requirements.txt ` .
requirements.txt created from it .

Comment (marcoooo) [reply requirements.txt:73]: so https should come from previous then.

**Review by marcoooo — DISMISSED** [review]

**Review by marcoooo — APPROVED** [review]

**Pull request merged**